app/controllers/admin/users_controller.rb

```diff
@@ -24,19 +24,26 @@ def update
 
 
   def enable
-    User.find(params[:id]).update(disabled: false)
+    @user = User.find(params[:id])
+    @user.update(disabled: false)
     flash[:primary] = 'You have enabled a user'
-    redirect_to admin_users_path
+    redirect_helper
   end
 
   def disable
-    User.find(params[:id]).update(disabled: true)
+    @user = User.find(params[:id])
+    @user.update(disabled: true)
     flash[:primary] = 'You have disabled a user'
-    redirect_to admin_users_path
+    redirect_helper
   end
 
   private
 
+  def redirect_helper
```

Comment:
Nice helper method, I like how you were able to use the same code for different user types

Reply:
I agree!